smartsim/_core/_cli/build.py

```diff
@@ -265,12 +261,13 @@ def build_redis_ai(
                     # we will always be able to find a torch version downloaded by
                     # pip so if we can't find it we know the user suggested a torch
                     # installation path that doesn't exist
-                    logger.error("Could not find requested user Torch installation")
-                    sys.exit(1)
+                    raise SetupError("Could not find requested user Torch installation")
```

Comment:
Do exceptions get captured into logfiles? If not, we may be losing the opportunity to share with users what went wrong

Reply:
If the message is getting logged, then I'm good. Removing sys.exit() calls seems like a good plan to me

Reply:
`+1 love it` to removing those sys.exit calls. i think of them like the "view" in MVC - they make it harder to test behavior alone